docs/Using Fleet/MDM-migration-guide.md

```diff
@@ -7,45 +7,173 @@ This section provides instructions for migrating your hosts away from your old M
+
+2. On your **My device** page, select **Turn on MDM** the button in the yellow banner and follow the instructions. 
+    * If you don’t see the yellow banner or the **Turn on MDM** button, select the purple **Refetch** button at the top of the page. 
+    * If you still don't see the **Turn on MDM** button or the **My device** page presents you with an error, please contact your IT administrator.
+
+<img width="1400" alt="My device page - turn on MDM" src="https://user-images.githubusercontent.com/5359586/229950406-98343bf7-9653-4117-a8f5-c03359ba0d86.png">
+
+#### End user workflow
 
 > Available in Fleet Premium or Ultimate
 
-You can use Fleet's end user migration workflow to reduce the gap in MDM coverage during migration.
+The end user migration workflow is supported for automatically enrolled (DEP) hosts.
+
+To watch a GIF that walks through the end user experience for the end user migration workflow, in the Fleet UI, head to **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+
+In Fleet, you can configure the end user workflow using the Fleet UI or fleetctl command-line tool.
+
+Fleet UI:
+
+1. Select the avatar on the right side of the top navigation and select **Settings > Integrations > Mobile device management (MDM)**.
 
-The migration worfklow is supported for automatically enrolled (DEP) hosts.
+2. Scroll down to the **End user migration workflow** section and select the toggle to enable the workflow.
 
-During the end user migration workflow, an end user's device will have their selected system
-theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds,
-you can optionally set an alternate logo for the themes.
+3. Under **Mode** choose a mode and enter the webhook URL for you automation tool (ex. Tines) under **Webhook URL** and select **Save**.
 
-You can do this in the Fleet UI by going to **Settings** > **Organization settings** >
-**Organization info** and adding a url to the desired image in the **Organization avatar URL (for
-dark backgrounds)** and **Organization avatar URL (for light backgrounds)** inputs. The appropriate
-image will show depending on the selected system theme.
+4. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds, you can optionally set a logo for each theme:
+
+    1. Head to **Settings** > **Organization settings** >
+**Organization info** and add URLs to your logos in the **Organization avatar URL(for dark backgrounds)** and **Organization avatar URL (for light backgrounds)** fields.
+    
+    2. Select **Save**.
+
+fleetctl CLI:
+
+1. Create `fleet-config.yaml` file or add to your existing `config` YAML file:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  mdm:
+    macos_migration:
+      enable: true
+      mode: "voluntary"
```

Comment:
I think we should omit quotes if there aren't special characters that need escaping, since I don't see us using them in other examples

Reply:
Hmm, I think we need quotes for "voluntary" in this example because `mode` expects a string. @gillespi314 is that right?

Reply:
My understanding is that YAML strings only need quotes when escaping certain special characters. Here's the team config as an example, many strings aren't in quotes:
![Screenshot 2023-08-15 at 6 09 42 PM](https://github.com/fleetdm/fleet/assets/3065949/9ae456fd-30cd-4c81-b164-abe9f8abb48a)


That said, don't want this to block merging since that syntax isn't _incorrect_— just thinking about conventions for these examples going forward.